nodestream/model/graph_objects.py

```diff
@@ -18,22 +22,42 @@ def update(self, other: "DeduplicatableObject"):
         raise NotImplementedError
 
 
+# This is a weird little trick to cache the current time for a short period of
+# time. When nodestream is adding the timestamp to tones of node and
+# relationship objects, it can be a bit slow. Just to get times that are
+# basically the same, there is little value in getting the current time over
+# and over again. So we cache the current time for a short period of time.
+# Profiling this generally shows a 5-15% speedup in the time it takes to
+# ingest a large number of nodes and relationships (Excluding database time).
+
+
+@lru_cache(maxsize=128)
+def _get_cached_timestamp(time_hash: int) -> Timestamp:
+    del time_hash  # Unused; just needed to cache the result
+    return Timestamp.utcnow()
```

Comment:
I wonder if maybe we can just use datetime.utcnow() and then print it with the right format?

Reply:
Not sure I fully understand

Reply:
I wasn't sure how this was used. But if we can use `datetime.utcnow().timestamp()` AND that is faster than Timestamp.utcnow(), maybe we could use that.

Stackoverflow is telling me that `datetime.utcnow()` is magnitudes faster